.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,31 @@
+name: build_and_test
+
+on:
+  push:
+  pull_request:
+  repository_dispatch:
+    types: [backend_automation]
+  workflow_dispatch:
+
+jobs:
+  build_and_test:
```

Comment:
Let's name it in the way the other workflows - mention in it the idea of build in short.